lib/JsonLdDocumentLoader.js

```diff
@@ -18,8 +19,50 @@ class JsonLdDocumentLoader {
     this.documents.set(url, document);
   }
 
+  /**
+   * Adds a custom protocol handler to the loader.
+   *
+   * @example
+   * // Using did-io's CachedResolver instance as a did: protocol handler
+   * jld.addProtocolHandler({protocol: 'did', handler: cachedResolver});
+   *
+   * @param {string} protocol - Protocol id, such as 'did', 'https', etc.
+   * @param {{get: Function}} handler - Protocol handler object, exposes an
+   *   async `get({url})` method.
+   */
+  addProtocolHandler({protocol, handler} = {}) {
+    this.protocolHandlers.set(protocol, handler);
+  }
+
+  /**
+   * Adds a DID method specific driver to protocol handlers.
+   *
+   * @param {object} didMethodDriver - A did-io style did method driver, such
+   *   as did-method-key and did-veres-one. (Exposes a `.method` property,
+   *   and a `.get()` function.)
+   */
+  addDidResolver(didMethodDriver) {
```

Comment:
Seems a little strange to call the param a `didMethodDriver` when it handles the whole `did` protocol as opposed to a particular method. It seems like someone using this function shouldn't pass it a did method driver, but a full DID resolver that itself may be composed of DID method drivers for individual methods. Can we update the docs to reflect that?

```suggestion
  addDidResolver(didResolver) {
```

Reply:
Changed.